pkg/loghelper/loghelper.go

```diff
@@ -18,23 +18,45 @@ package loghelper
 
 import (
 	"fmt"
+	"path"
+	"runtime"
+	"strconv"
+	"time"
 
 	"github.com/sirupsen/logrus"
 )
 
 // CuratedLogLines
 type CuratedLogLines struct {
-	lines []string
+	Lines []string
```

Comment:
I believe this can remain lowercase/private.  We have `GetLogLines` to get the value from the object.

Reply:
I needed it to be able to initialize the logs as below: 
wantClaimsLog: loghelper.CuratedLogLines{Lines: []string{"IPv4 ping test on network default from ( container: test1 pod: test-0 ns: tnf  srcip: 10.244.195.231 ) to ( container: test2 pod: test-1 ns: tnf dstip: 10.244.195.232 ) result: outcome: FAILURE transmitted: 10 received: 5 errors: 5\n", //nolint:lll
				"IPv4 ping test on network default from ( container: test1 pod: test-0 ns: tnf  srcip: 10.244.195.231 ) to ( container: test3 pod: test-1 ns: tnf dstip: 10.244.195.233 ) result: outcome: FAILURE transmitted: 10 received: 5 errors: 5\n"}}, //nolint:lll
		},
I'll think about other ways, maybe an init function ?

Reply:
Ah yeah, I had to do the same thing here: https://github.com/test-network-function/cnf-certification-test/blob/main/cnf-certification-test/lifecycle/graceperiod/graceperiod_test.go#L119-L125

Reply:
fixed, added init function